ci/travis/before_deploy.sh

```diff
@@ -32,13 +34,15 @@ if [ "${TRAVIS_BRANCH}" = "master" ]; then
     pip install --upgrade pip
 
     # Assuming pip 1.5.X is installed.
-    echo "pip install wheel --user"
     pip install wheel --user
 
+    cd ${TRAVIS_BUILD_DIR}/bindings/py
+
     # Build all NuPIC and all required python packages into dist/wheels as .whl
     # files.
-    echo "pip wheel --wheel-dir=dist/wheels ."
-    pip wheel --wheel-dir=dist/wheels .
+    pip wheel --wheel-dir=dist/wheels -r requirements.txt
```

Comment:
This is in nupic.core.  nupic.bindings isn't in requirements.txt because these are the requirements for nupic.bindings